<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisuAlea</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/openalea_icon.png</normaloff>:/icons/openalea_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabPackager">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="packageview">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/package.png</normaloff>:/icons/package.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Package</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="categoryview">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/category.png</normaloff>:/icons/category.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Category</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="searchview">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Search</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="search_lineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTabWidget" name="poolTabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pooltab">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/datapool.png</normaloff>:/icons/datapool.png</iconset>
         </attribute>
         <attribute name="title">
          <string>DataPool</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabWorkspace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="usetab">
         <attribute name="title">
          <string>Root</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <widget class="QMenu" name="menuApplication">
      <property name="title">
       <string>Application</string>
      </property>
      <addaction name="actionPreview_Application"/>
      <addaction name="actionExport_to_Application"/>
     </widget>
     <addaction name="menuApplication"/>
     <addaction name="actionTo_script"/>
     <addaction name="separator"/>
     <addaction name="action_Image"/>
     <addaction name="action_Svg"/>
    </widget>
    <widget class="QMenu" name="menuLast_open">
     <property name="title">
      <string>Open recent</string>
     </property>
    </widget>
    <addaction name="action_New_Empty_Workspace"/>
    <addaction name="menuLast_open"/>
    <addaction name="separator"/>
    <addaction name="action_Export_to_Factory"/>
    <addaction name="action_Close_current_workspace"/>
    <addaction name="actionReload_from_Model"/>
    <addaction name="menu_Export"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuDataPool">
    <property name="title">
     <string>&amp;DataPool</string>
    </property>
    <addaction name="actionClear_Data_Pool"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
    <addaction name="actionOpenAlea_Web"/>
   </widget>
   <widget class="QMenu" name="menu_Python">
    <property name="title">
     <string>P&amp;ython</string>
    </property>
    <addaction name="action_Execute_script"/>
    <addaction name="actionOpen_Console"/>
    <addaction name="actionClea_r_Console"/>
   </widget>
   <widget class="QMenu" name="menu_Workspace">
    <property name="windowIcon">
     <iconset>
      <normaloff/>
     </iconset>
    </property>
    <property name="title">
     <string>&amp;Workspace</string>
    </property>
    <widget class="QMenu" name="menuAlign">
     <property name="title">
      <string>Align</string>
     </property>
     <addaction name="actionAlignHorizontally"/>
     <addaction name="actionAlignLeft"/>
     <addaction name="actionAlignRight"/>
     <addaction name="actionAlignMean"/>
     <addaction name="actionDistributeHorizontally"/>
     <addaction name="actionDistributeVertically"/>
    </widget>
    <widget class="QMenu" name="menuColour">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionSetCustomColor"/>
     <addaction name="actionUseCustomColor"/>
    </widget>
    <addaction name="action_Run"/>
    <addaction name="actionInvalidate"/>
    <addaction name="actionReset"/>
    <addaction name="actionConfigure_I_O"/>
    <addaction name="actionRecordProvenance"/>
    <addaction name="separator"/>
    <addaction name="actionGroup_Selection"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Delete_2"/>
    <addaction name="separator"/>
    <addaction name="menuAlign"/>
    <addaction name="menuColour"/>
   </widget>
   <widget class="QMenu" name="menu_Package">
    <property name="title">
     <string>&amp;Package Manager</string>
    </property>
    <widget class="QMenu" name="menuCreate">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <addaction name="actionNew_Package"/>
     <addaction name="action_New_Network"/>
     <addaction name="actionNew_Python_Node"/>
     <addaction name="action_Data_File"/>
    </widget>
    <addaction name="action_Add_File"/>
    <addaction name="action_Auto_Search"/>
    <addaction name="actionFind_Node"/>
    <addaction name="separator"/>
    <addaction name="menuCreate"/>
    <addaction name="separator"/>
    <addaction name="actionShow_log"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionDisplay_Package_Manager"/>
    <addaction name="actionDisplay_Workspaces"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Package"/>
   <addaction name="menuDataPool"/>
   <addaction name="menu_Workspace"/>
   <addaction name="menu_Python"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_New_Package">
   <property name="text">
    <string>&amp;Create Package</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSystem_Search">
   <property name="text">
    <string>System &amp;Search</string>
   </property>
  </action>
  <action name="action_Add_File">
   <property name="text">
    <string>&amp;Load Package/Directory</string>
   </property>
  </action>
  <action name="action_Auto_Search">
   <property name="text">
    <string>&amp;Reload All packages</string>
   </property>
  </action>
  <action name="action_Close_current_workspace">
   <property name="text">
    <string>Close Current Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_New_Network">
   <property name="text">
    <string>&amp;Composite Node</string>
   </property>
  </action>
  <action name="actionOpenAlea_Web">
   <property name="text">
    <string>OpenAlea Web</string>
   </property>
  </action>
  <action name="action_Execute_script">
   <property name="text">
    <string>&amp;Execute script</string>
   </property>
  </action>
  <action name="action_New_Session">
   <property name="text">
    <string>&amp;New Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open_Session">
   <property name="text">
    <string>&amp;Open Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_Session">
   <property name="text">
    <string>&amp;Save Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as</string>
   </property>
  </action>
  <action name="action_Export_to_Factory">
   <property name="text">
    <string>&amp;Save as Composite Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionExport_to_Application">
   <property name="text">
    <string>Export to Application</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionClear_Data_Pool">
   <property name="text">
    <string>Clear Data Pool</string>
   </property>
  </action>
  <action name="actionFind_Node">
   <property name="text">
    <string>&amp;Find Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNew_Python_Node">
   <property name="text">
    <string>Python &amp;Node</string>
   </property>
  </action>
  <action name="actionNew_Package">
   <property name="text">
    <string>&amp;Package</string>
   </property>
  </action>
  <action name="action_EditNode">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="actionShow_Pool">
   <property name="text">
    <string>Show Pool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_OpenNode">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Delete_2">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_New_Empty_Workspace">
   <property name="text">
    <string>&amp;New Empty Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionReload_from_Model">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionConfigure_I_O">
   <property name="text">
    <string>Configure I/O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionRecordProvenance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record Provenance</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionGroup_Selection">
   <property name="text">
    <string>Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_Console">
   <property name="text">
    <string>Open &amp;shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPreview_Application">
   <property name="text">
    <string>Preview Application</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLeft_Panel">
   <property name="text">
    <string>Package Manager</string>
   </property>
  </action>
  <action name="actionWorkspaces">
   <property name="text">
    <string>Workspaces</string>
   </property>
  </action>
  <action name="actionDisplay_Package_Manager">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Package Manager</string>
   </property>
  </action>
  <action name="actionDisplay_Workspaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Workspaces</string>
   </property>
  </action>
  <action name="actionInvalidate">
   <property name="text">
    <string>Invalidate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionClea_r_Console">
   <property name="text">
    <string>&amp;Clear shell</string>
   </property>
  </action>
  <action name="action_Image">
   <property name="text">
    <string>&amp;Image</string>
   </property>
  </action>
  <action name="action_Svg">
   <property name="text">
    <string>&amp;Svg</string>
   </property>
  </action>
  <action name="action_Application">
   <property name="text">
    <string>&amp;Application</string>
   </property>
  </action>
  <action name="action_Data_File">
   <property name="text">
    <string>&amp;Data File</string>
   </property>
  </action>
  <action name="actionShow_log">
   <property name="text">
    <string>Show log</string>
   </property>
  </action>
  <action name="actionAlignHorizontally">
   <property name="text">
    <string>Align Horizontally</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="text">
    <string>Align Left</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="text">
    <string>Align Right</string>
   </property>
  </action>
  <action name="actionAlignMean">
   <property name="text">
    <string>Align Mean</string>
   </property>
  </action>
  <action name="actionSetCustomColor">
   <property name="text">
    <string>Set Custom Color...</string>
   </property>
   <property name="toolTip">
    <string>Set Custom Colour</string>
   </property>
  </action>
  <action name="actionUseCustomColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Custom Color</string>
   </property>
   <property name="toolTip">
    <string>Use Custom Color</string>
   </property>
  </action>
  <action name="actionDistributeHorizontally">
   <property name="text">
    <string>Distribute Horizontally</string>
   </property>
  </action>
  <action name="actionDistributeVertically">
   <property name="text">
    <string>Distribute Vertically</string>
   </property>
   <property name="toolTip">
    <string>DistributeVertically</string>
   </property>
  </action>
  <action name="actionTo_script">
   <property name="text">
    <string>Script</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionGf">
   <property name="text">
    <string>gf</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
